server/utils/checkYourAnswersUtils.ts

```diff
@@ -1,12 +1,21 @@
 /* eslint-disable import/prefer-default-export */
-import { ApprovedPremisesApplication } from '@approved-premises/api'
+import { ApprovedPremisesApplication, ApprovedPremisesAssessment } from '@approved-premises/api'
 import { HtmlItem, SummaryListItem, Task, TextItem } from '@approved-premises/ui'
 import Apply from '../form-pages/apply'
 import paths from '../paths/apply'
 
 import { getResponseForPage } from './applicationUtils'
 
-const checkYourAnswersSections = (application: ApprovedPremisesApplication) => {
+const checkYourAnswersSections = (application: ApprovedPremisesApplication) =>
+  reviewSections(application, getTaskResponsesAsSummaryListItems)
+
+const reviewSections = (
```

Comment:
I agree with the gist of what I think you're saying (that shared functions should be in shared files) but I'm hesitant to name a file to do with the assessment review stage 'checkYourAnswers' as I think it might get confusing if the two pages diverge later down the line.